src/rules/converters/space-within-parens.ts

```diff
@@ -0,0 +1,14 @@
+import { RuleConverter } from "../converter";
+
+export const convertSpaceWithinParens: RuleConverter = tslintRule => {
+    const arg = tslintRule.ruleArguments.length === 1 ? "always" : "never";
+    return {
+        rules: [
+            {
+                ruleArguments: [arg],
+                ruleName: "@typescript-eslint/space-within-parens",
+                notices: ["The number of spaces will be ignored"],
```

Comment:
Ah, almost - this notice isn't relevant if the user has configured to `never` allow spaces, right? As in, the rule behavior then is the same in the two linters. It's just when there are spaces in play that they differ?

Reply:
So should the notice only show when parameters are passed then?

Reply:
I believe so, yeah.